pages/my-reports/script.js

```diff
@@ -0,0 +1,95 @@
+import MyReport from '../../assets/helpers/report-card.js'
+import { API_URL } from '../../constants.js'
+import { getUserSession } from '../../assets/helpers/storage.js'
+
+const user = getUserSession()
+let userID = user?.id
+let recentReportArr = []
+let olderReportArr = []
+const recentReports = document.getElementById('recentReports')
+const olderReports = document.getElementById('olderReports')
+const recentBtn = document.getElementById('recentReportsBtn')
+const olderBtn = document.getElementById('olderReportsBtn')
+
+// Checkbox toggle
+// Set the recentBtn to be checked initially
+recentBtn.checked = true
+displayRecentReports()
+displayOlderReports()
```

Comment:
It would be good to trigger the older reports request only when it's needed, when the user clicks on the tab

Reply:
done https://github.com/whamester/WMDD-4885-Project/pull/24/commits/320f9e479e312154914ac3d6c01732655307dbc1